crates/fdev/src/commands.rs

```diff
@@ -71,10 +65,12 @@ async fn put_contract(
     params: Parameters<'static>,
 ) -> Result<(), anyhow::Error> {
     let contract = ContractContainer::try_from((config.code.as_path(), params))?;
-    let state = {
+    let state = if let Some(ref state_path) = contract_config.state {
         let mut buf = vec![];
-        File::open(&contract_config.state)?.read_to_end(&mut buf)?;
+        File::open(state_path)?.read_to_end(&mut buf)?;
         buf.into()
+    } else {
+        vec![].into()
```

Comment:
let's add a warning here at least, is ok to send empty states if that's well handled by the contract itself but we should warn the developer at least

Reply:
Add a warn, but GitHub cannot show the changes...